src/main/java/game/pojo/Order.java

```diff
@@ -0,0 +1,14 @@
+package game.pojo;
+
+public abstract class Order {
+    Country destination;
+    int armyNumber;
+
+    public Order(Country destination, int armyNumber) {
```

Comment:
In the future, other orders will take different parmeters, so I think we can just have the default construtor for the Order class (it doesn't really matter anyway since it's abstract and it can just be fixed after build 1)

Reply:
Otherwise I don't really have anything to say, code looks ok to me

Reply:
I thought about that too but also most orders like advance and attack will require such parameters too